.github/workflows/test_fast.yml

```
@@ -20,11 +20,17 @@ jobs:
       fail-fast: false # Don't let a failed MacOS run stop the Ubuntu runs
       matrix:
         os: ['ubuntu-latest']
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
+        python-version: ['3.7', '3.8', '3.10', '3.11']
         include:
-          - os: 'macos-11'
-            python-version: '3.7'
+          # mac os test
+          - os: 'macos-latest'
```

I think the merge of 8.2.x into this branch wasn't quite right
```suggestion
          - os: 'macos-11'
```